DataFormats/Detectors/MUON/MCH/include/DataFormatsMCH/Digit.h

```diff
@@ -50,15 +50,15 @@ class Digit
   int getPadID() const { return mPadID; }
   void setPadID(int padID) { mPadID = padID; }
 
-  unsigned long getADC() const { return mADC; }
-  void setADC(unsigned long adc) { mADC = adc; }
+  uint32_t getADC() const { return mADC; }
+  void setADC(uint32_t adc) { mADC = adc; }
 
  private:
   int32_t mTFtime;      /// time since the beginning of the time frame, in bunch crossing units
   uint16_t mNofSamples; /// number of samples in the signal
   int mDetID;           /// ID of the Detection Element to which the digit corresponds to
   int mPadID;           /// PadIndex to which the digit corresponds to
-  unsigned long mADC;   /// Amplitude of signal
+  uint32_t mADC;        /// Amplitude of signal
 
   ClassDefNV(Digit, 2);
```

Comment:
while we don't really use that information so far as far as I know, I would still increase the classdef with any format change, for consistency.

Reply:
Indeed I completely forgot.. Fixed.